README.md

```diff
@@ -0,0 +1,19 @@
+# Project Title
+Authiclay
+
+## Project Description
+A small sized authenticator system using pure NodeJs, that acts as a 2 Factor authentication for web applications.
+
+## Prerequisites
+Node Js
+
+## The Process
```

Comment:
You don't need this process, yet. You can remove the content in this section.

Reply:
Done.